back/app/services/base_side_fx_service.rb

```diff
@@ -0,0 +1,23 @@
+class BaseSideFxService
```

Comment:
Interesting idea. It would be also good if we could somehow derive the sideFx class name from the controller name. But it seems it's not possible in our case, which means we should add a new method called `side_fx_class_name`.

Also, it's currently pretty convenient to go to definitions of classes like `SideFxCampaignService`. It's just one click in the controller in VSCode. But if we use @sidefx variable, it would be a bit longer.

So, right now I'm not sure about it.